src/server/BaseHttpHandler.ts

```diff
@@ -0,0 +1,72 @@
+import type { ErrorHandler } from '../ldp/http/ErrorHandler';
+import type { RequestParser } from '../ldp/http/RequestParser';
+import type { ResponseDescription } from '../ldp/http/response/ResponseDescription';
+import type { ResponseWriter } from '../ldp/http/ResponseWriter';
+import type { Operation } from '../ldp/operations/Operation';
```

Comment:
Are all operations LDP operations? Or should this one move to the generic folder too?